src/ui/public/courier/fetch/call_response_handlers.js

```diff
@@ -26,13 +26,13 @@ export function CallResponseHandlersProvider(Private, Promise) {
   const ABORTED = RequestStatus.ABORTED;
   const INCOMPLETE = RequestStatus.INCOMPLETE;
 
-  function callResponseHandlers(requests, responses) {
-    return Promise.map(requests, function (req, i) {
-      if (req === ABORTED || req.aborted) {
+  function callResponseHandlers(searchRequests, responses) {
+    return Promise.map(searchRequests, function (searchRequest, index) {
+      if (searchRequest === ABORTED || searchRequest.aborted) {
         return ABORTED;
       }
 
-      const resp = responses[i];
+      const resp = responses[index];
```

Comment:
Should resp be response here to be consistent?  Looks like response is used elsewhere.